query/src/sessions/session_settings.rs

```diff
@@ -300,7 +300,7 @@ impl Settings {
     }
 
     fn set_to_global(&self, setting: &mut SettingValue) -> Result<()> {
-        let tenant = self.session_ctx.get_tenant();
```

Comment:
how about remove the `get_tenant()` method of session_ctx struct? 

newcomers may miss the different uses between the two methods 🤔